src/main/java/uk/gov/justice/digital/domain/DomainExecutor.java

```diff
@@ -0,0 +1,271 @@
+package uk.gov.justice.digital.domain;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.domain.model.*;
+import uk.gov.justice.digital.domain.model.TableDefinition.TransformDefinition;
+import uk.gov.justice.digital.domain.model.TableDefinition.ViolationDefinition;
+import uk.gov.justice.digital.service.DataStorageService;
+import uk.gov.justice.digital.service.DomainOperations;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.*;
+
+import uk.gov.justice.digital.exception.DomainExecutorException;
+
+public class DomainExecutor extends DomainOperations {
```

Comment:
As we discussed yesterday, `DomainExecutor` should not extend `DomainOperations`. Instead make `DomainOperations` a singleton and have micronaut pass it in via DI.